docs/creating-templates.md

```diff
@@ -8,4 +8,35 @@ Coming Soon!
 2. [Extending your bot using packages](/docs/extending-with-packages.md)
 3. [Extending your bot with code](/docs/extending-with-code.md)
 4. [Creating your own packages](/docs/creating-packages.md)
-5. [Creating your own templates](/docs/creating-templates.md)
\ No newline at end of file
+5. [Creating your own templates](/docs/creating-templates.md)
+
+## Template Publishing
+
+Template packages indicate to Composer which azure environments and runtime languages it supports through the keywords associated with the package. 
+
+The following feedUrl is used by Composer to grab the first party template generators.
+- If user has not opted in to using preview generators
+  - https://registry.npmjs.org/-/v1/search?text=generator+keywords:bf-template+scope:microsoft+maintainer:botframework
```

Comment:
@pavolum this query isn't coming back with populated `objects`. After some testing, this url might address what you're trying to achieve: https://registry.npmjs.org/-/v1/search?text=generator+maintainer:botframework+keywords:bf-template

Reply:
Thanks for catching, have made the change